src/main/java/com/revealprecision/revealserver/messaging/message/MetaDataEvent.java

```diff
@@ -0,0 +1,24 @@
+package com.revealprecision.revealserver.messaging.message;
+
+import com.revealprecision.revealserver.persistence.domain.metadata.infra.TagData;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
```

Comment:
@Data also contains @Getter @Setter, you can remove some here depending on your goal